pkgs/development/python-modules/loca/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, pythonOlder, fetchFromSourcehut }:
+
+buildPythonPackage rec {
+  pname = "loca";
+  version = "2.0.1";
+  format = "flit";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromSourcehut {
+    owner = "~cnx";
+    repo = pname;
+    rev = version;
+    sha256 = "1l6jimw3wd81nz1jrzsfw1zzsdm0jm998xlddcqaq0h38sx69w8g";
+  };
+
+  doCheck = false;
```

Comment:
Why did you disable tests?

Reply:
[The only checks are static analyses](https://builds.sr.ht/~cnx/job/560833), i.e. linting, so I thought it would be redundant to rerun them.

Reply:
Could you leave this as a comment then?

Reply:
Done!